docker/docker-compose.yml

```diff
@@ -2,103 +2,72 @@ version: '2'
 
 services:
 
-  se-source:
-    image: busybox
-    container_name: se-source
-    volumes:
-      - ./../:$SRC_DIR # code volume
-      - /opt/media/shopelectro/:$SRC_DIR/media/
-      # leaved it here as example of code mounting for debugging
-      # - $REFARM_SRC/search:/root/.local/lib/python3.6/site-packages/search
-    command: sh -c "top > /dev/null"
-
-  se-python:
-    build:
-      context: ..
-      dockerfile: docker/python/dev/Dockerfile
+  app:
     image: fidals/se:dev
-    container_name: se-python
+    env_file:
+      - env_files/app
+      - env_files/credentials
+      - env_files/paths
+      - env_files/ports
     environment:
       - ALLOWED_HOSTS=*
       - TEST_ENV=true
-      - DJANGO_SETTINGS_MODULE=$DJANGO_SETTINGS_MODULE
-      - DJANGO_LOG_LEVEL=$DJANGO_LOG_LEVEL
-      - DATABASE_URL=postgres://$DB_USER:$DB_PASS@se-postgres/$DB_NAME
-      - BROCKER_URL=amqp://$RABBITMQ_DEFAULT_USER:$RABBITMQ_DEFAULT_PASS@se-rabbitmq:5672/
-      - REDIS_PASSWORD=$REDIS_PASSWORD
-      - REDIS_LOCATION_DEFAULT=redis://se-redis:6379/0
-      - REDIS_LOCATION_SESSION=redis://se-redis:6379/1
-      - REDIS_LOCATION_THUMBNAIL=redis://se-redis:6379/2
-      - REDIS_LOCATION_USER_AGENT=redis://se-redis:6379/3
-      - FTP_USER=$FTP_USER
-      - FTP_PASS=$FTP_PASS
-      - FTP_IP=$FTP_IP
-      - SECRET_KEY=$SECRET_KEY
-      - YANDEX_SHOP_PASS=$YANDEX_SHOP_PASS
-      - EMAIL_HOST_PASSWORD=$EMAIL_HOST_PASSWORD
-      - ENV_TYPE=$ENV_TYPE
+      - REDIS_LOCATION_DEFAULT=redis://redis:6379/0
```

Comment:
let's put it to some env file. Maybe seprated: `env_files/urls` / `env_files/service_urls` or smth.

Now it's duplicated in compose files

Reply:
@artemiy312 , what about this comment? Review fix doesn't cover it